packages/tracing-internal/src/browser/request.ts

```diff
@@ -275,22 +275,19 @@ export function xhrCallback(
   }
 
   const scope = getCurrentScope();
-  const parentSpan = getActiveSpan();
-
-  const span =
-    shouldCreateSpanResult && parentSpan
-      ? // eslint-disable-next-line deprecation/deprecation
-        parentSpan.startChild({
-          data: {
-            type: 'xhr',
-            'http.method': sentryXhrData.method,
-            url: sentryXhrData.url,
-          },
-          description: `${sentryXhrData.method} ${sentryXhrData.url}`,
-          op: 'http.client',
-          origin: 'auto.http.browser',
-        })
-      : undefined;
+
+  const span = shouldCreateSpanResult
+    ? startInactiveSpan({
+        attributes: {
+          type: 'xhr',
+          'http.method': sentryXhrData.method,
+          url: sentryXhrData.url,
```

Comment:
just to make sure: Do these attribute keys need `sentry.` prefixes? 

If not, is there a source where we define which attributes need a prefix vs which don't? I assume it's somewhere [here](https://github.com/getsentry/sentry-conventions/blob/main/model/trace/sentry/sentry.yaml)?

Reply:
These attribute keys do not need `sentry.X` because they are not specific to sentry product or schema. The conventions you linked are the list so far!